test/test_nn.py

```diff
@@ -3728,6 +3728,13 @@ def forward(self, *args):
         module_name='PoissonNLLLoss',
         input_size=(2, 3, 4, 5),
         target_fn=lambda: torch.randn(2, 3, 4, 5).floor_().abs_(),
+        desc='non_full_loss',
```

Comment:
To be consistent with the other loss function tests, the desc should be 'no_reduce', so the full test name is `test_PoissonNLLLoss_no_reduce`